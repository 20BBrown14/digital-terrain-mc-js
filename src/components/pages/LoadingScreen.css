.loading-screen-container {
  position: fixed;
  width: 100%;
  height: 100%;
  display: flex;
  align-items: center;
  top: 0;
  right: 0
}

.loading-screen-element {
  margin: 0 auto;
}