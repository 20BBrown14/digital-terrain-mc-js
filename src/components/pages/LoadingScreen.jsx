import React from 'react';
import 'spinkit/spinkit.min.css';
import './LoadingScreen.css';

const loadingScreen = () => (
  <div className="loading-screen-container">
    <div className="sk-chase loading-screen-element">
      <div className="sk-chase-dot" />
      <div className="sk-chase-dot" />
      <div className="sk-chase-dot" />
      <div className="sk-chase-dot" />
      <div className="sk-chase-dot" />
      <div className="sk-chase-dot" />
    </div>
  </div>
);

export default loadingScreen;
