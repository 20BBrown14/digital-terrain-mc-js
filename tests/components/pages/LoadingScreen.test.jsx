import React from 'react';
import LoadingScreen from '../../../src/components/pages/LoadingScreen';

describe('LoadingScreen', () => {
  it('renders a default view', () => {
    const loadingScreen = shallow(<LoadingScreen />);
    expect(loadingScreen).toMatchSnapshot();
  });
});
