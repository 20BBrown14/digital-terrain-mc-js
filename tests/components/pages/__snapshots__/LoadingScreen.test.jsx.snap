// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingScreen renders a default view 1`] = `
<div
  className="loading-screen-container"
>
  <div
    className="sk-chase loading-screen-element"
  >
    <div
      className="sk-chase-dot"
    />
    <div
      className="sk-chase-dot"
    />
    <div
      className="sk-chase-dot"
    />
    <div
      className="sk-chase-dot"
    />
    <div
      className="sk-chase-dot"
    />
    <div
      className="sk-chase-dot"
    />
  </div>
</div>
`;
